manifests/init.pp

```diff
@@ -117,12 +133,16 @@
 # @author Trevor Vaughan <tvaughan@onyxpoint.com>
 #
 class simp (
+  # This parameter set from data in modules
   Hash                            $scenario_map,
-  Enum['simp',
-       'simp_lite',
-       'poss',
-       'none',
-       'remote_access']           $scenario                   = 'simp',
+  Enum[
```

Comment:
@heliocentric Correction. Since the docs are now updated, the current function makes sense and I have reverted this back to `String`.